code/modules/surgery/organs/lavaland.dm

```diff
@@ -0,0 +1,255 @@
+		for(var/mob/living/L in listeners)
+			var/throwtarget = get_edge_target_turf(owner, get_dir(owner, get_step_away(L, owner)))
+			L.throw_at_fast(throwtarget, 3, 1)
+		next_command = world.time + cooldown_damage
+
+	//WHO ARE YOU?
+	else if(findtext(command, "who are you?") || findtext(command, "say your name") || findtext(command, "state your name") || findtext(command, "identify"))
+		for(var/mob/living/L in listeners)
+			L.say("[L.real_name]")
+		next_command = world.time + cooldown_meme
+
+	//MOVE
+	else if(findtext(command, "move"))
+		for(var/mob/living/L in listeners)
+			var/turf/T = get_step(L,pick(cardinal))
+			L.Move(T)
+		next_command = world.time + cooldown_meme
+
+	//WALK
+	else if(findtext(command, "walk") || findtext(command, "slow down"))
+		for(var/mob/living/L in listeners)
+			L.m_intent = MOVE_INTENT_WALK
+		next_command = world.time + cooldown_meme
+
+	//RUN
+	else if(findtext(command, "run"))
+		for(var/mob/living/L in listeners)
+			L.m_intent = MOVE_INTENT_RUN
+		next_command = world.time + cooldown_meme
+
+	//HELP INTENT
+	else if(findtext(command, "help"))
+		for(var/mob/living/carbon/human/H in listeners)
+			H.a_intent_change(INTENT_HELP)
+		next_command = world.time + cooldown_meme
+
+	//DISARM INTENT
+	else if(findtext(command, "disarm"))
+		for(var/mob/living/carbon/human/H in listeners)
+			H.a_intent_change(INTENT_DISARM)
+		next_command = world.time + cooldown_meme
+
+	//GRAB INTENT
+	else if(findtext(command, "grab"))
+		for(var/mob/living/carbon/human/H in listeners)
+			H.a_intent_change(INTENT_GRAB)
+		next_command = world.time + cooldown_meme
+
+	//HARM INTENT
+	else if(findtext(command, "harm") || findtext(command, "fight"))
+		for(var/mob/living/carbon/human/H in listeners)
+			H.a_intent_change(INTENT_HARM)
+		next_command = world.time + cooldown_meme
+
+	//THROW/CATCH
+	else if(findtext(command, "throw") || findtext(command, "catch"))
+		for(var/mob/living/carbon/C in listeners)
+			C.throw_mode_on()
+		next_command = world.time + cooldown_meme
+
+	//FLIP
+	else if(findtext(command, "flip") || findtext(command, "rotate") || findtext(command, "revolve") || findtext(command, "roll") || findtext(command, "somersault"))
+		for(var/mob/living/L in listeners)
+			L.emote("flip")
+		next_command = world.time + cooldown_meme
+
+	//REST
+	else if(findtext(command, "rest"))
+		for(var/mob/living/L in listeners)
+			L.resting = TRUE
+		next_command = world.time + cooldown_meme
+
+	//GET UP
+	else if(findtext(command, "get up") || findtext(command, "stand"))
+		for(var/mob/living/L in listeners)
+			L.resting = FALSE
+			L.SetWeakened(0)
+			L.SetParalysis(0)
+		next_command = world.time + cooldown_damage
+
+	//SIT
+	else if(findtext(command, "sit"))
+		for(var/mob/living/L in listeners)
+			for(var/obj/structure/chair/chair in get_turf(L))
+				chair.buckle_mob(L)
+				break
+		next_command = world.time + cooldown_meme
+
+	//STAND UP
+	else if(findtext(command, "stand"))
```

Comment:
stand is also used in stun, thus would never run.  I'd suggest simply putting this else-if above the stun one.